src/pages/studyView/UserSelections.tsx

```diff
@@ -451,6 +493,21 @@ export default class UserSelections extends React.Component<
                               )
                             : this.renderDataBinFilter(
                                   clinicalDataFilter.values,
+                                  async (
```

Comment:
why is it async?

Reply:
oh Im sure thats from before I refactored pulling the actual asynchronous code out into `updateCustomIntervalFilter`